CONTRIBUTING.md

```diff
@@ -17,9 +18,15 @@ Once it is approved for merging, you should bump the versioning (this repository
 - `make bump-minor`
 - `make bump-patch`
 
-Push the bump commit into the GitHub PR.
+Push the bump commit into the GitHub PR.  
+**Please ensure to push tags as well: `git push --tags` IF you do not `followTags = true` in your ~/.gitconfig**  
```

Comment:
PRs don't include tags so i think even if you push them up to github merging the PR still won't cause them to be applied

https://stackoverflow.com/a/12279290/521361

Reply:
True, this is regarding the conversation we had that I should add tags.
Another thing is that maybe this info would be useful for people pushing straight to master.

You decide if to remove it